exercises/K-functions-parameters/exercise3.js

```diff
@@ -1,5 +1,7 @@
 // Write your function here
+function greeting(name){
+ console.log("Hello,my name is ;"+ name);
```

Comment:
I think here it would have make more sense based on the exercise to return the greeting and then use console.log() outside of the function.

Reply:
function greeting(name){
return name;
}
console.log("Hello,my name is ;"+greeting("Daniel"));


Is it ok?